celery_utils/management/commands/drop_djcelery_tables.py

```diff
@@ -0,0 +1,80 @@
+"""
+This command will drop the tables included in django-celery's
+migrations, which are included when we bump that to 3.2.1
+
+If the tables are not empty, this command will show a message and exit.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import OrderedDict
+import logging
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from djcelery.models import (CrontabSchedule, IntervalSchedule, PeriodicTask,  # pylint:disable=import-error
+                             PeriodicTasks, TaskMeta, TaskSetMeta, TaskState, WorkerState)
+
+log = logging.getLogger(__name__)
+
+DJCELERY_MODEL_TABLES = OrderedDict({
+    PeriodicTask: 'djcelery_periodictask',  # must be before crontab, interval
+    TaskState: 'djcelery_taskstate',  # must be before workerstate
+})
+DJCELERY_MODEL_TABLES.update({
```

Comment:
Are you sure? I though that, since it's a dict, the initial parameter's order was not necessarily preserved:

```
>>> from collections import OrderedDict
>>> a = OrderedDict({"z": 10, "b": 5, "a": 1})
>>> a
OrderedDict([('a', 1), ('z', 10), ('b', 5)])
>>> a.values()
[1, 10, 5]
>>> a.keys()
['a', 'z', 'b']
```